test/NuGet.Clients.FuncTests/NuGet.CommandLine.FuncTest/Commands/PushCommandTest.cs

```diff
@@ -1,18 +1,25 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using NuGet.CommandLine.Test;
-using NuGet.Test.Utility;
 using System;
 using System.IO;
 using System.Net;
 using System.Threading;
+using NuGet.CommandLine.Test;
+using NuGet.Test.Utility;
 using Xunit;
 
 namespace NuGet.CommandLine.FuncTest.Commands
 {
     public class PushCommandTest
     {
+        private const string MESSAGE_EXISTING_PACKAGE = "already exists at feed"; //Derived from resx: AddPackage_PackageAlreadyExists
```

Comment:
Where are these resources put? 

If you add internals visible to for this assembyl from what I presume is NuGet.Commands, you'd be able to use the exact string and avoid test changes if the strings ever change.

Reply:
Existing tests follow a hard-coded pattern. I just made constants so at least it's only one place to change.

Reply:
I understand that, but it doesn't make them great :)